### ISSUE TITLE
This project is abandoned by the Developer and Customer Support promised with purchase of license is non-existent.

### ISSUE THREAD

**Issue by Ryyval** [Live-Charts/Live-Charts#946]
How to reproduce this issue: Purchase licenses, and then email support with questions in need of help.

You won't receive a response.

**Comment by hunterzzzpro** [comment]
Yeah, this is technically illegal, no support as promised, old bugs a year later still not resolved and in a state of "in progress". 

I have checked the authors' StackOverflow and he is often on there but not contributing. 

If anyone buys this library get a chargeback refund as it looks like he is taking the money and running!

**Comment by MoSalih1** [comment]
Hi guys, 
I've implemented a solver using this library but it's very slow. I am running out of time and need to purchase a license. I just need the fast rendering, do you think it will work fine?
